projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx

```diff
@@ -96,12 +119,36 @@ const FeaturePromptComponent = props => {
 			} ) }
 			answer={
 				<div className="jp-recommendations-question__install-section">
-					<Button primary rna href={ nextRoute } onClick={ onInstallClick }>
-						{ ctaText }
-					</Button>
-					<a href={ nextRoute } onClick={ onDecideLaterClick }>
-						{ __( 'Not now', 'jetpack' ) }
-					</a>
+					{ featureActive ? (
+						<>
+							<div className="jp-recommendations-question__feature-enabled">
+								<div className="jp-recommendations-question__checkmark">
+									<Gridicon icon="checkmark-circle" size={ 24 } />
+								</div>
+								<span>Feature Enabled</span>
```

Comment:
This string should be translatable.

Reply:
Looks like this missing translation still needs to be addressed, Josh.

Reply:
Ack. I remember fixing it... maybe I reset this file at some point in the process 🙃 
Thanks for catching that!